modules/jdbc/src/main/java/org/apache/ignite/internal/jdbc/JdbcResultSet.java

```diff
@@ -265,7 +307,7 @@ public boolean next() throws SQLException {
     /** {@inheritDoc} */
     @Override
     public void close() throws SQLException {
-        close0();
+        closed = true;
```

Comment:
all resources must be cleaned up as soon as possible, it's not enough to just pretend that result set is closed

Reply:
done